src/silly_sprites/opengl/window.cpp

```diff
@@ -36,19 +36,19 @@ namespace sly::gl {
         GLFWwindow* const window = glfwCreateWindow(width, height, "coder2k bester Mann", nullptr, nullptr);
         if (window == nullptr) {
             spdlog::critical("Failed to create GLFW window");
-            return tl::unexpected{ GlError::FailedToCreateWindow };
+            throw GLError{ GLErrorType::FailedToCreateWindow };
         }
         glfwMakeContextCurrent(window);
         glfwSetInputMode(window, GLFW_STICKY_KEYS, GLFW_TRUE);
         glfwSetInputMode(window, GLFW_STICKY_MOUSE_BUTTONS, GLFW_TRUE);
 
         if (not gladLoadGL(glfwGetProcAddress)) {
             spdlog::critical("Failed to initialize Glad");
-            return tl::unexpected{ GlError::FailedToInitializeGlad };
+            throw GLError{ GLErrorType::FailedToInitializeGlad };
         }
 
-        on_framebuffer_size_changed(window, width, height);
-        glfwSetFramebufferSizeCallback(window, on_framebuffer_size_changed);
+        gl::on_framebuffer_size_changed(window, width, height);  // addressing the gl namespace becuase otherwise the function is ambiguous
```

Comment:
```suggestion
        gl::on_framebuffer_size_changed(window, width, height);  // addressing the gl namespace because otherwise the function is ambiguous
```